### DISCUSSION TITLE
Chat Estimate: Option to use Placed token's Token Name instead of Prototype

### DISCUSSION THREAD

**Discussion by King-Wedgie** [mclemente/healthEstimate#148] [Ideas]
Title? My specific use case would be for compatibility with PF2E Workbench's NPC Mystification function; it works by replacing a placed token's Token Name with creature attributes until it's demystified.
![image](https://user-images.githubusercontent.com/73143074/230930745-72b44175-b80c-4b36-b587-f2bd05522ec4.png)

**Comment by mclemente** [comment]
Deleted my reply because it wasn't correct.

It can be done. The easiest way would be for Workbench to set a flag that the current token is mystified. Otherwise, there's no guarantee the token's name was specifically changed by it and it may lead to unexpected behavior.

**Comment by King-Wedgie** [comment]
The new NPC Mystification support works perfectly, thanks!
